sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/utils/string-utils.ts

```diff
@@ -0,0 +1,34 @@
+import { QueueProcessSummaryResult } from "../../../../../../../processors/models/processors.models";
+
+/**
+ * Converts a number to its fixed text format.
+ * @param value value to be converted.
+ * @param options options.
+ * - `length` the fixed length. Default 10.
```

Comment:
There is no `options` parameter in the method

Reply:
Just changed it, please take a look.